exporter/signalfxexporter/factory_test.go

```diff
@@ -315,9 +315,8 @@ func TestCreateMetricsExporterWithEmptyExcludeMetrics(t *testing.T) {
 
 func testMetricsData() pmetric.Metrics {
```

Comment:
One last thing. Do we really need changes to this test case? It looks like it just breaks the metrics into two ResourceMetrics objects starting from `m4` which is not desired I think

Reply:
The goal was for the test metrics to have the same format that they'd have if they were coming from the `hostmetrics` receiver. I broke them into two `ResourceMetrics` objects to differentiate the time, showing that the two objects were coming one after the other, since you wouldn't get the same metric twice, at two different timestamps, within the same `ScopeMetric` object, or `ResourceMetric`.

I could at least update the variable naming to make it more clear? I think the changes should stay though so at least the test is more future proofed for changes.

Reply:
Ok makes sense now!